rectype,issueid,project_owner,project_name,actor,time,text,action,title,clean_text
rectype,issueid,project_owner,project_name,actor,time,text,action,title,text
issue_title,993,nilearn,nilearn,salma1601,2016-02-17 18:54:05,"On my labtop this test runs OK with sudo privileges and fails without sudo

May be it is because it relies on fake atlas with 5 labels created for this test but without sudo privileges data is not created. Then `data_dir` for the function `fetch_atlas_harvard_oxford` is `None` so it loads my real atlas in FSL DIR which has 97 labels.

Traceback (most recent call last):
File ""/usr/lib/python2.7/dist-packages/nose/case.py"", line 197, in runTest
self.test(*self.arg)
File ""/home/salma/CODE/Parietal/nilearn/nilearn/datasets/tests/test_atlas.py"", line 143, in test_fail_fetch_atlas_harvard_oxford
assert_equal(len(ho.labels), 5)
AssertionError: 97 != 5
'97 != 5' = '%s != %s' % (safe_repr(97), safe_repr(5))
'97 != 5' = self._formatMessage('97 != 5', '97 != 5')
raise self.failureException('97 != 5')
",start issue,test_atlas.test_fail_fetch_atlas_harvard_oxford fails without sudo privileges,On labtop test run OK sudo privileg fail without sudo may reli fake atla 5 label creat test without sudo privileg data creat then function load real atla fsl dir 97 label traceback recent call last file usrlibpython27distpackagesnosecasepi line 197 runtest selftestselfarg file homesalmacodeparietalnilearnnilearndatasetsteststestatlaspi line 143 testfailfetchatlasharvardoxford assertequallenholabel 5 assertionerror 97 5 97 5 saferepr97 saferepr5 97 5 selfformatmessage97 5 97 5 rais selffailureexception97 5
issue_closed,993,nilearn,nilearn,AlexandreAbraham,2016-02-18 09:44:53,,closed issue,test_atlas.test_fail_fetch_atlas_harvard_oxford fails without sudo privileges,
issue_comment,993,nilearn,nilearn,AlexandreAbraham,2016-02-18 09:19:29,"Fixed in 2d806ec1507b4f7b5cd742aedb71d8467b58c572

Can you pull master and check?
",,,fix 2d806ec1507b4f7b5cd742aedb71d8467b58c572 can pull master check
issue_comment,993,nilearn,nilearn,salma1601,2016-02-18 09:40:27,"Yes it's OK now, thanks !
",,,ye OK thank
issue_comment,993,nilearn,nilearn,AlexandreAbraham,2016-02-18 09:44:53,"Cool. Thanks for reporting!
",,,cool thank report
